src/ptwt/conv_transform_2.py

```diff
@@ -3,6 +3,7 @@
 The implementation relies on torch.nn.functional.conv2d and
 torch.nn.functional.conv_transpose2d under the hood.
 """
+# Written by the Pytorch wavelet toolbox team in 2024
```

Comment:
I don't think these are necessary, the license for the repo covers this

Reply:
(also, this makes the code more difficult to read)

Reply:
I took these out.